### PR TITLE
pybind/mgr: PEP 8 code clean and fix typo

### PR DESCRIPTION
**Pull request by iliul**
- Code clean report by pycharm
- Fix typo

Signed-off-by: Lei Liu <liul.stone@gmail.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/pybind/mgr/mgr_module.py [hunk 1]
```diff
@@ -160,7 +161,9 @@ def get_pools_by_take(self, take):
         return self._get_pools_by_take(take).get('pools', [])
 
     def calc_pg_upmaps(self, inc,
-                       max_deviation=.01, max_iterations=10, pools=[]):
+                       max_deviation=.01, max_iterations=10, pools=None):
```
Comment (tchaikov) [inline src/pybind/mgr/mgr_module.py]: 👍

File: src/pybind/mgr/prometheus/module.py [hunk 2]
```diff
@@ -598,8 +597,8 @@ def get_rbd_stats(self):
         pools_refreshed = False
         if pools:
             next_refresh = self.rbd_stats['pools_refresh_time'] + \
-                self.get_localized_module_option(
-                    'rbd_stats_pools_refresh_interval', 300)
+                           self.get_localized_module_option(
```
Comment (tchaikov) [inline src/pybind/mgr/prometheus/module.py]: what is this change for?

File: src/pybind/mgr/prometheus/module.py [hunk 3]
```diff
@@ -86,8 +85,8 @@ def __init__(self, mtype, name, desc, labels=None):
         self.mtype = mtype
         self.name = name
         self.desc = desc
-        self.labelnames = labels    # tuple if present
-        self.value = {}             # indexed by label values
+        self.labelnames = labels  # tuple if present
```
Comment (tchaikov) [inline src/pybind/mgr/prometheus/module.py]: what is this change for?

Comment (iliul) [reply src/pybind/mgr/prometheus/module.py]: ok

File: src/pybind/mgr/prometheus/module.py (hunk 2 above)
Comment (dmick) [reply src/pybind/mgr/prometheus/module.py]: indentation; to line up the right side of the =

File: src/pybind/mgr/prometheus/module.py (hunk 3 above)
Comment (dmick) [reply src/pybind/mgr/prometheus/module.py]: "ok" as in "I'll remove the change", or?...

Comment (dmick) [reply src/pybind/mgr/prometheus/module.py]: I think the self.value init is required

**Comment by dmick** [comment]
LGTM except for self.value above

File: src/pybind/mgr/prometheus/module.py (hunk 3 above)
Comment (iliul) [reply src/pybind/mgr/prometheus/module.py]: @dmick Yes, I had remove the change, `self.value` is initialize at [here](https://github.com/iliul/ceph/blob/8b868fec151d30937cf37cb5042dc7be2dd25773/src/pybind/mgr/prometheus/module.py#L89)

File: src/pybind/mgr/prometheus/module.py (hunk 2 above)
Comment (iliul) [reply src/pybind/mgr/prometheus/module.py]: I'll remove the change

Comment (dmick) [reply src/pybind/mgr/prometheus/module.py]: I think it's correct, though, no?

Comment (tchaikov) [reply src/pybind/mgr/prometheus/module.py]: @dmick yes, it's correct. but i don't think we should encourage this kind of reformatting changes. as it is not specified by a guideline or a PEP AFAICT.

@iliul first, i don't think it is required by PEP8.

and i guess the current indention is what i get with my Emacs with default (well, mostly) settings. and probably is what other developers will have with their editors.

if we **really** care about the indentation here. i'd suggest add quote the expression at the right side of `=` with parentheses, like
```py
            next_refresh = (self.rbd_stats['pools_refresh_time'] +
                            self.get_localized_module_option(
                                'rbd_stats_pools_refresh_interval', 300))
```

i don't feel strong either way. but i still suggest drop this change, personally.

Comment (dmick) [reply src/pybind/mgr/prometheus/module.py]: I'll defer to deeper research; I didn't read PEP8, I just assumed the change was suggested in compliance with PEP8.  Whatever.

Comment (tchaikov) [reply src/pybind/mgr/prometheus/module.py]: we can run `pep8` or `pycodestyle` against this source file. presumably, it should find out all PEP8 issues.

Comment (iliul) [reply src/pybind/mgr/prometheus/module.py]: @tchaikov OK, Use [`autopep8`](https://github.com/hhatto/autopep8) checked again, command is:
```shell
# autopep8 --in-place prometheus/module.py mgr_module.py mgr_util.py hello/module.py
```

Only the files related to this PR, others should be put into the next job( NEW PR).

**Comment by tchaikov** [comment]
```
  3/162 Test   #6: run-tox-mgr-insights ....................***Failed    1.64 sec
ERROR: tox version is 2.3.1, required is at least 2.8.1
```

on arm64 . not related to this change. just wanted note it down and make it more visible.

**Comment by iliul** [comment]
> @iliul lgtm. but in future, i'd recommend to accompany the reformatting changes along with non-formatting-only changes like refactor, performance improvement or bug fix. as massive reformatting could make it more difficult to track the changes of source code.
> 
> normally, we do formatting changes when we happen to work on related code.

Got it, Thank you

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2019-02-05_14:55:38-rados-wip-kefu2-testing-2019-02-05-1807-distro-basic-smithi/

**Pull request merged**